app/workers/reactions/bust_reactable_cache_worker.rb

```diff
@@ -0,0 +1,20 @@
+module Reactions
+  class BustReactableCacheWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(reaction_id, cache_buster = CacheBuster)
```

Comment:
In a similar PR not so long ago we removed the default arg and inlined it.  As @mstruve rightfully pointed out, if we ever specify something other than the default, it would need to be string that we later `constantize` in order to survive the roundtrip to Sidekiq . Maybe we should do the same here and just change the spec to

```ruby
before do
  allow(CacheBuster).to receive(:bust)
end
```

Reply:
+1

Reply:
@citizen428 @rhymes Could you guys look at this [discussion](https://github.com/thepracticaldev/dev.to/pull/5581#discussion_r368291722)? I had to do the same change, but spec is failing in this PR